.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "ocurrent"]
 	path = ocurrent
-	url = https://github.com/ocurrent/ocurrent.git
+	url = https://github.com/dra27/ocurrent.git
```

Comment:
Note: you shouldn't need to change this; it's enough that a PR is open on the main repository with the commit.

Reply:
It'll get upset if I rebase the upstream PR, though, right?

Reply:
(either way, it'll be blown away before any merge!)